Auth: Add support for OIDC RP-Initiated Logout (#70357)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/_index.md

### signout_redirect_url

> URL to redirect the user to after they sign out. To support [OpenID Connect RP-Initiated Logout](https://openid.net/specs/openid-connect-rpinitiated-1_0.html) , user need to add `post_logout_redirect_uri` to the `signout_redirect_url`.

Reviewer: ```suggestion
The URL the user is redirected to upon signing out. To support [OpenID Connect RP-Initiated Logout](https://openid.net/specs/openid-connect-rpinitiated-1_0.html), the user must add `post_logout_redirect_uri` to the `signout_redirect_url`.
```

> Eg. signout_redirect_url = http://localhost:8087/realms/grafana/protocol/openid-connect/logout?post_logout_redirect_uri=http%3A%2F%2Flocalhost%3A3000%2Flogin

Reviewer: ```suggestion
Example: 

signout_redirect_url = http://localhost:8087/realms/grafana/protocol/openid-connect/logout?post_logout_redirect_uri=http%3A%2F%2Flocalhost%3A3000%2Flogin
```

> > **Note**: `post_logout_redirect_uri` parameter needs to be given after the openid-connect logout url.

Reviewer: ```suggestion
{{% admonition type="note" %}}
The `post_logout_redirect_uri` parameter must be given after the openid-connect logout url.
{{% /admonition %}}
```

Reviewer: What do we mean by the word "given"?

Author: I meant that post_logout_redirect_uri should be put after the logout url.
